torch/utils/tensorboard/summary.py

```diff
@@ -73,6 +74,48 @@ def _draw_single_box(image, xmin, ymin, xmax, ymax, display_str, color='black',
     return image
 
 
+def hparams(hparam_dict=None, metric_dict=None):
+
+    PLUGIN_NAME = 'hparams'
+    PLUGIN_DATA_VERSION = 0
+
+    EXPERIMENT_TAG = '_hparams_/experiment'
```

Comment:
Seems like these hardcoded values would be an issue?

Reply:
They are copied from tensorboard/tensorboard/plugins/hparams/metadata.py I don't think other functions in that file is useful here. The plugin will have no function if these constants are changed in the future version of tensorboard. (And the legacy output of tensorflow will have a similar situation.)

ps. I used this kind of porting for many plugins before.